project-base/storefront/components/Basic/SymfonyToolbar/SymfonyToolbar.tsx

```diff
@@ -0,0 +1,13 @@
+import { isEnvironment } from 'helpers/isEnvironment';
+import { useDomainConfig } from 'hooks/useDomainConfig';
+import Script from 'next/script';
+
+export const SymfonyToolbar: FC = () => {
+    const { url } = useDomainConfig();
+
+    if (!isEnvironment('development')) {
```

Comment:
How do I turn this feature off as a developer? I say it should be possible to completely turn off (by ENV or whatever). I am not sure about other devs, but I personally do not want it there 😆